### DISCUSSION TITLE
Installation issues due to unsupported ros2 distribution

### DISCUSSION THREAD

**Discussion by 2ManySmells** [osrf/rmf_core#317] [Q&A]
Hi!

I recently decided to dabble with rmf_core/ros2, however I found out that there are some discrepancies in the installation process.

ROS2 currently does not support (EOL) eloquent turtle of ROS2 anymore, this is evident during the installation process in terminal.

![image](https://user-images.githubusercontent.com/73010613/111291325-ad349000-8679-11eb-81ae-0b319c82c458.png)

However, the installation instructions for rmf_core suggests that it requires eloquent. I tried side-stepping/ignoring the potential issue by substituting the commands provided.

- This was the command provided by rmf_core documentation for installation:
```
source /opt/ros/eloquent/setup.bash
```

- This is the command we used instead:
```
source /opt/ros/foxy/setup.bash
```

This is the error that's thrown after attempting full installation of rmf_core:
![image](https://user-images.githubusercontent.com/73010613/111291780-1d431600-867a-11eb-8f96-55c753127d22.png)

Is there anything that we can do to get a successful installation of rmf_core? I am making an assumption that there is issues with the installation instructions/or there are other steps i should take since i have a different distribution of ros2. 

Appreciate the help :)

**Comment by mxgrey** [comment]
Sorry about that! This is a fast-moving project, and unfortunately the documentation tends to lag behind the current actual state. RMF is currently targeting the Foxy distribution, not Eloquent.

The latest installation instructions can be found here: https://github.com/open-rmf/rmf

We are currently migrating all of our work on RMF to the `open-rmf` organization, which is where it will live for the long-term.

**Reply by 2ManySmells** [reply]
Thanks for the prompt reply!

Sounds good, i'll check out the repo :)
